edu.wpi.first.wpilib.plugins.cpp/resources/templates/iterative/Robot.cpp

```diff
@@ -2,16 +2,19 @@
 #include <memory>
 #include <string>
 
+#include <IterativeRobot.h>
 #include <LiveWindow/LiveWindow.h>
 #include <SmartDashboard/SendableChooser.h>
 #include <SmartDashboard/SmartDashboard.h>
 
-class Robot: public IterativeRobot {
+class Robot: public frc::IterativeRobot {
 public:
 	void RobotInit() {
-		chooser->AddDefault(autoNameDefault, static_cast<void*>(&autoNameDefault));
-		chooser->AddObject(autoNameCustom, static_cast<void*>(&autoNameCustom));
-		SmartDashboard::PutData("Auto Modes", chooser);
+		chooser.AddDefault(autoNameDefault,
+				const_cast<void*>(static_cast<const void*>(&autoNameDefault)));
```

Comment:
Not really. The static_cast could be replaced with a C-style cast (which is bad practice), but the const_cast is required. allwpilib#433 would be a better way to handle it.